src/pathfinding_service/service.py

```diff
@@ -352,13 +343,18 @@ def on_pfs_update(self, message: UpdatePFS) -> None:
             other_participant=other_participant,
         )
 
-        valid_nonces = (
+        known_nonces = (
             message.updating_nonce <= view_to_partner.update_nonce
             and message.other_nonce <= view_from_partner.update_nonce
         )
-        if valid_nonces:
+        if known_nonces:
```

Comment:
I would expect `known_nonces` to be a list of nonces. Maybe `are_nonces_known` or `is_any_nonce_new` (with the opposite value)?